src/azul/project/hca/__init__.py

```diff
@@ -312,3 +312,166 @@ def service_config(self) -> azul.plugin.ServiceConfig:
                 "protocol"
             ]
         )
+
+    def portal_integrations_db(self) -> JSON:
+        """
+        A hardcoded example database for use during development of the integrations API implementation
+        """
+        return [
+            {
+                "portal_id": "7bc432a6-0fcf-4c19-b6e6-4cb6231279b3",
+                "portal_name": "Terra Portal",
+                "portal_icon": "https://app.terra.bio/static/media/logo.c5ed3676.svg",
+                "contact_email": "",
+                "organization_name": "Terra",
+                "portal_description": "Terra is a cloud-native platform for biomedical researchers to access data, run analysis tools, and collaborate.",
+                "integrations": [
+                    {
+                        "integration_id": "b87b7f30-2e60-4ca5-9a6f-00ebfcd35f35",
+                        "integration_type": "get_manifest",
+                        "entity_type": "file",
+                        "title": "",
+                        "manifest_type": "full",
+                        "portal_url_template": "https://app.terra.bio/#import-data?url={manifest_url}"
+                    }
+                ]
+            },
+            {
+                "portal_id": "9852dece-443d-42e8-869c-17b9a86d447e",
+                "portal_name": "Single Cell Portal",
+                "portal_icon": "https://portals.broadinstitute.org/single_cell/assets/SCP-white-74aae87e22d39d3abee9531b5f1a0a177a5ac5be4c69cf9949d705877cb0a246.png",
+                "contact_email": "",
+                "organization_name": "Broad Institute",
+                "portal_description": "Reducing barriers and accelerating single-cell research.",
+                "integrations": [
+                    {
+                        "integration_id": "977854a0-2eea-4fec-9459-d4807fe79f0c",
+                        "integration_type": "get",
+                        "entity_type": "project",
+                        "title": "",
```

Comment:
We should fill in value for this property. The title should describe what the user gets if they activate this integrations. Consider asking the Broad person who sent us these links.